sfge/src/main/java/com/salesforce/rules/PathBasedRuleRunner.java

```diff
@@ -140,6 +142,8 @@ private void executeRulesOnPaths(List<ApexPath> paths) {
                         // objects.
                         if (ruleThrowable instanceof FlsViolationInfo) {
                             incompleteThrowables.add((FlsViolationInfo) ruleThrowable);
+                        } else if (ruleThrowable instanceof MassSchemaLookupInfo) {
+                            tempDeleteMe.add((MassSchemaLookupInfo) ruleThrowable);
```

Comment:
Needs a minor refactor to clean up. Also, there seems to be a bug with multiple rule types. I'll address this in another commit.